### PR TITLE
feat: remove axios in favor of fetch API

### PR DESCRIPTION
**Pull request by fgiuliani**
BREAKING CHANGE: axios is no longer a dependency

### CODE REVIEW

**Review by alexjoverm — APPROVED** [review]

File: lib/index.ts [hunk 1]
```diff
@@ -7,10 +7,10 @@ import {
 import {
```
Comment (manuelschroederdev) [inline lib/index.ts:7]: general question: shouldn't it be `import type`?

**Review by manuelschroederdev — APPROVED** [review]

File: lib/types.ts [hunk 2]
```diff
@@ -10,27 +10,34 @@ export interface SbReactSDKOptions extends SbSDKOptions {
 }
 
 export type {
-  AlternateObject,
-  Richtext,
-  RichtextInstance,
+  apiPlugin,
```
Comment (alexjoverm) [inline lib/types.ts:13]: apiPlugin should be a normal export (not type)

File: lib/index.ts [hunk 3]
```diff
@@ -20,15 +20,16 @@ export {
   useStoryblokBridge,
   registerStoryblokBridge,
   renderRichText,
+  RichTextResolver,
```
Comment (alexjoverm) [inline lib/index.ts:23]: Well done!

**Review by alexjoverm — CHANGES_REQUESTED** [review]

File: lib/types.ts (hunk 2 above)
Comment (Dawntraoz) [reply lib/types.ts:13]: @alexjoverm this is also exported as a type in the Vue 3 SDK, could this be the cause of the apiPlugin issue?

Comment (fgiuliani) [reply lib/types.ts]: @alexjoverm changed!

The normal export is here:
https://github.com/storyblok/storyblok-react/blob/main/lib/index.ts#L19

Comment (alexjoverm) [reply lib/types.ts]: @fgiuliani perfect!
@Dawntraoz not really. If you remember from the last call we know where the problem is.

**Review by alexjoverm — APPROVED** [review]

**Pull request merged**